packages/bench/package.json

```diff
@@ -8,24 +8,24 @@
     "type-check": "tsc"
   },
   "dependencies": {
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0",
+    "vue": "^3.4.27",
+    "vue-router": "^4.3.2"
+  },
```

Comment:
We must handle dependencies correctly, otherwise oxlint will report the following error:
```bash
  ⚠ eslint-plugin-import(named): named import "createApp" not found
   ╭─[packages\bench\vue-entry.js:1:10]
 1 │ import { createApp } from 'vue'
   ·          ─────────
 2 │ import { createRouter, createWebHistory } from 'vue-router'   
   ╰────
  help: does "vue" have the export "createApp"?
```